sfepy/discrete/common/fields.py

```diff
@@ -4,9 +4,10 @@
 import numpy as nm
 
 from sfepy.base.base import output, iter_dict_of_lists, Struct, basestr
-from sfepy.base.multiproc import use_multiprocessing, managers
+import sfepy.base.multiproc as multi
```

Comment:
Try moving this import into the functions that need it to break the hard dependence on mpi.